app/src/main/java/com/mxt/anitrend/util/NotificationUtil.kt

```diff
@@ -46,13 +47,15 @@ class NotificationUtil(
         )
     }
 
-    private fun dismissNotificationsIntent(): PendingIntent {
-        val onDismissIntent = Intent(this.context, DismissNotification::class.java)
+    private fun clearNotificationsIntent(action: String): PendingIntent {
+        val intent = Intent(this.context, ClearNotifications::class.java)
+        intent.putExtra(KeyUtil.NOTIFICATION_ID, defaultNotificationId)
+        intent.putExtra(KeyUtil.NOTIFICATION_ACTION, action)
         return PendingIntent.getBroadcast(
-            this.context,
-            0,
-            onDismissIntent,
-            0
+            context,
+            action.hashCode(),
```

Comment:
With request code hardcoded as 0 I would always receive dismiss action in broadcast receiver thus I'm passing the hashcode here.

Reply:
That should be fine 💯